spec/features/user_edit_spec.rb

```diff
@@ -1,77 +1,95 @@
 require 'rails_helper'
 
 describe 'edit staff users' do
-  let!(:user) { create :user, staff: true, last_name: 'Smith' }
+  context 'with admin privilege' do
+    let!(:user) { create :user, staff: true, last_name: 'Smith' }
```

Comment:
I don't know, this is from the previous person to have made this. However, I can freely delete it and see if some tests fail as a result?